app/src/main/java/com/iic/mokojin/ChooseCharactersActivity.java

```diff
@@ -197,6 +197,7 @@ private void refreshMenu(){
 
         private void updateTitle() {
             ChooseCharactersActivity charactersActivity = (ChooseCharactersActivity) getActivity();
+            if (null == charactersActivity) return;
```

Comment:
I'm not sure, I just got an exception because of that when moving quickly between screens